strftime and semver devDependencies are required at runtime

[These two dependencies](https://github.com/CartoDB/Windshaft-cartodb/blob/master/package.json#L43) are required at runtime when I launch app.js.

I haven't had the chance to investigate further but moving these two under dependencies and running `npm install --no-shrinkwrap` worked around the issue

----
```javascript

module.js:340
    throw err;
    ^
Error: Cannot find module 'semver'
    at Function.Module._resolveFilename (module.js:338:15)
    at Function.Module._load (module.js:280:25)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at Object.<anonymous> (/bb/cartodb/windshaft-cartodb/embedded/lib/node_modules/windshaft-cartodb/node_modules/windshaft/node_modules/grainstore/lib/grainstore/style_trans.js:1:76)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Module.require (module.js:364:17)

module.js:340
    throw err;
    ^
Error: Cannot find module 'strftime'
    at Function.Module._resolveFilename (module.js:338:15)
    at Function.Module._load (module.js:280:25)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at Object.<anonymous> (/bb/cartodb/windshaft-cartodb/embedded/lib/node_modules/windshaft-cartodb/node_modules/cartodb-redis/lib/carto_metadata.js:10:19)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Module.require (module.js:364:17)
````

I am getting the following error while running `npm shrinkwrap`

npm WARN unmet dependency /bb/var/omnibus/src/Windshaft-cartodb/node_modules/windshaft requires semver@'~1.1.4' but will load
npm WARN unmet dependency undefined,
npm WARN unmet dependency which is version undefined
npm WARN unmet dependency /bb/var/omnibus/src/Windshaft-cartodb/node_modules/cartodb-redis requires strftime@'~0.8.2' but will load
npm WARN unmet dependency undefined,
npm WARN unmet dependency which is version undefined
npm WARN unmet dependency /bb/var/omnibus/src/Windshaft-cartodb/node_modules/windshaft/node_modules/grainstore requires semver@'~1.1.4' but will load
npm WARN unmet dependency undefined,
npm WARN unmet dependency which is version undefined
npm ERR! Error: Problems were encountered
npm ERR! Please correct and try again.
npm ERR! missing: istanbul@~0.3.6, required by windshaft-cartodb@2.7.1
npm ERR! missing: mocha@~1.21.4, required by windshaft-cartodb@2.7.1
npm ERR! missing: nock@~1.3.0, required by windshaft-cartodb@2.7.1
npm ERR! missing: jshint@~2.6.0, required by windshaft-cartodb@2.7.1
npm ERR! missing: redis@~0.8.6, required by windshaft-cartodb@2.7.1
npm ERR! missing: strftime@~0.8.2, required by windshaft-cartodb@2.7.1
npm ERR! missing: semver@~1.1.4, required by windshaft-cartodb@2.7.1
npm ERR! missing: semver@~1.1.4, required by windshaft@0.47.0
npm ERR! missing: semver@~1.1.4, required by grainstore@1.0.0
npm ERR! missing: strftime@~0.8.2, required by cartodb-redis@0.13.0
npm ERR!     at shrinkwrap_ (/bb/datavis/cartodb/embedded/lib/node_modules/npm/lib/shrinkwrap.js:30:15)
npm ERR!     at /bb/datavis/cartodb/embedded/lib/node_modules/npm/lib/shrinkwrap.js:24:5
npm ERR!     at /bb/datavis/cartodb/embedded/lib/node_modules/npm/lib/ls.js:45:30
npm ERR!     at /bb/datavis/cartodb/embedded/lib/node_modules/npm/node_modules/read-installed/read-installed.js:130:5
npm ERR!     at /bb/datavis/cartodb/embedded/lib/node_modules/npm/node_modules/read-installed/read-installed.js:244:14
npm ERR!     at cb (/bb/datavis/cartodb/embedded/lib/node_modules/npm/node_modules/slide/lib/async-map.js:47:24)
npm ERR!     at /bb/datavis/cartodb/embedded/lib/node_modules/npm/node_modules/read-installed/read-installed.js:244:14
npm ERR!     at cb (/bb/datavis/cartodb/embedded/lib/node_modules/npm/node_modules/slide/lib/async-map.js:47:24)
npm ERR!     at /bb/datavis/cartodb/embedded/lib/node_modules/npm/node_modules/read-installed/read-installed.js:244:14
npm ERR!     at cb (/bb/datavis/cartodb/embedded/lib/node_modules/npm/node_modules/slide/lib/async-map.js:47:24)
npm ERR! If you need help, you may report this *entire* log,
npm ERR! including the npm and node versions, at:
npm ERR!     <http://github.com/npm/npm/issues>

npm ERR! System Linux 3.10.0-229.el7.x86_64
npm ERR! command "/bb/datavis/cartodb/embedded/bin/node" "/bb/datavis/cartodb/embedded/bin/npm" "shrinkwrap"
npm ERR! cwd /bb/var/omnibus/src/Windshaft-cartodb
npm ERR! node -v v0.10.33
npm ERR! npm -v 1.4.28
npm ERR!
npm ERR! Additional logging details can be found in:
npm ERR!     /bb/var/omnibus/src/Windshaft-cartodb/npm-debug.log
npm ERR! not ok code 0

--- a/NEWS.md
+++ b/NEWS.md
@@ -4,6 +4,9 @@
 
 Released 2016-mm-dd
 
+Announcements:
+ - Upgrades cartodb-redis to 0.13.1.
+
 
 ## 2.64.0
 

--- a/package.json
+++ b/package.json
@@ -23,7 +23,7 @@
     "camshaft": "0.36.0",
     "cartodb-psql": "~0.6.1",
     "cartodb-query-tables": "~0.1.0",
-    "cartodb-redis": "~0.13.0",
+    "cartodb-redis": "0.13.1",
     "debug": "~2.2.0",
     "dot": "~1.0.2",
     "express": "~4.13.3",

